src/objects.rs

```diff
@@ -167,16 +175,20 @@ pub unsafe trait HasId: RoomObjectProperties {
 impl_has_id! {
     ConstructionSite,
     Creep,
+    Deposit,
```

Comment:
`Deposit` doesn't have an id in the API. Though it has one in the game... Interesting.